.github/workflows/main.yml

```diff
@@ -1,26 +1,64 @@
 name: Build and deploy jekyll site
-
 on:
   push:
     branches:
       - release
-      # It is highly recommended that you only run this action on push to a
-        # specific branch, eg. master or source (if on *.github.io repo)
-
 jobs:
-  jekyll:
+  build_and_deploy_site:
     permissions:
       contents: write
-    runs-on: ubuntu-16.04 # can change this to ubuntu-latest if you prefer
+    runs-on: ubuntu-latest
```

Comment:
Did that work for you?

Reply:
It works in that the workflow runs successfully. I can't see whether the site is deployed properly I suppose because I'd have to change some configuration files to deploy it on my repo.

In fact if I tried to use ubuntu-16-04 I got errors saying it was deprecated.

Reply:
Hmm. For me ubuntu-latest have an error.

Reply:
Yep, just tested it again and the workflow fails with ubuntu-16.04. Po4a won't work properly and the jekyll build fails.